Readme.md

```diff
@@ -60,15 +62,17 @@ cdb PV len : Current length of the PV for this position in cdb in plies.
 score      : The standard minimax score found, not using the decay that cdb implements.
 PV         : Best line found.
 PV len     : Length of this PV line in plies.
+max ply    : deepest line searched so far
 queryall   : Number of positions visited in the search tree, with results provided by cdb or the local cache.
 bf         : Branching factor q^(1/d) computed from queryall q and depth d.
-inflight   : Number of active/concurrent requests made to cdb on average.
+inflightR  : Number of active/concurrent requests made to cdb on average.
+inflightQ  : Number of active/concurrent queries made to cdb on average.
```

Comment:
Maybe replace `active/concurrent` just with `concurrent`or some other wording avoiding the `/` = "per" Interpretation? And is difference between `requests` and `queries` obvious here?

Reply:
done.